src/components/views/right_panel/RoomSummaryCard.tsx

```diff
@@ -171,7 +171,7 @@ const AppRow: React.FC<IAppRowProps> = ({ app, room }) => {
         { canModifyWidget && <ContextMenuTooltipButton
             className={classNames({
                 "mx_RoomSummaryCard_app_options": true,
-                "mx_RoomSummaryCard_maximised_widget": SettingsStore.getValue("feature_maximised_widgets"),
+                "mx_RoomSummaryCard_maximised_widget": true,
```

Comment:
the classNames lib is no longer needed here, just use a string `"mx_RoomSummaryCard_app_options mx_RoomSummaryCard_maximised_widget"`